packages/common-sdk/src/web3/transactions/transactions-builder.ts

```diff
@@ -397,9 +399,13 @@ export class TransactionBuilder {
       recentBlockhash = await this.connection.getLatestBlockhash(blockhashCommitment);
     }
     let finalComputeBudgetOption = computeBudgetOption ?? { type: "none" };
+    let margin = 0.1;
+
     if (finalComputeBudgetOption.type === "auto") {
-      const margin = finalComputeBudgetOption.computeLimitMargin ?? 0.1;
-      const lookupTableAccounts =
+      margin = finalComputeBudgetOption.computeLimitMargin ?? margin;
+    }
+
+    const lookupTableAccounts =
```

Comment:
This estimates computeBudgetLimit even if a static number is set in the `budgetOption`. Can we skip the estimation if a static number is set?

Reply:
updated so we only estimate the limit if `computeBudgetLimit` is not initially passed in for `fixed` type. checking for `undefined` to handle scenario where `0` is passed as value